SPECS/haproxy-dataplaneapi/haproxy-dataplaneapi.spec

```diff
@@ -0,0 +1,58 @@
+# https://fedoraproject.org/wiki/PackagingDrafts/Go
+%global commit          284d34bb4aebf36df8ba1ad430adfd72c222d08b
+%global commitdate      20200415
+%global shortcommit     %(c=%{commit}; echo ${c:0:7})
+%global repo            dataplaneapi
+%global cmd             %{repo}
```

Comment:
```suggestion
%global cmd             haproxy-%{repo}
```

Reply:
Hi @randomvariable,

I thought about that, but I think we want the binary name to be consistent with the name actually built upstream, don't we?

Reply:
i think it should make sense to a user running `ls` on the directory, in which case, maybe `%{_libexecdir}/haproxy/%{cmd}` where cmd is still dataplaneapi might combine both idioms?